requirements.txt

```diff
@@ -2,3 +2,4 @@ wrapanapi==3.5.10
 dynaconf==3.1.2
 wget==3.2
 click==7.1.2
+pre-commit==2.16.0
```

Comment:
latest is 2.19.0

Reply:
Fixed. Could you please review it again ? @jyejare